rhea/system/memmap/avalonmm.py

```diff
@@ -15,16 +15,18 @@
 
 
 class AvalonMM(MemoryMapped):
+
+    """AvalonMM bus object"""
+
     name = 'avalon'
 
     def __init__(self, glbl=None, data_width=8, address_width=16, name=None):
-        """ AvalonMM bus object
-        Parameters (kwargs):
-        --------------------
-        :param glbl: system clock and reset
-        :param data_width: data bus width
-        :param address_width: address bus width
-        :param name: name for the bus
+        """
```

Comment:
This docstring needs to be updated to the Google format outlined in sphinx-napolean:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/#google-vs-numpy

Some of the docstrings are old and I used the default format, but all docstrings should use the google format outlined in sphinx-napolean.  

I also use "Arguments" for the ports and "Parameters" for parameters, this might be an odd usage and not sure how the autodoc will extract but it is useful (very useful) to separate the ports/interfaces and the parameters.